style.css

```diff
@@ -0,0 +1,104 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  body {
+  
+      background: rgb(91,119,149);
+      background: linear-gradient(90deg, rgba(91,119,149,0.5717592592592593) 22%, rgba(55,197,209,0.3657407407407407) 63%, rgba(0,212,255,0.2222222222222222) 99%);
+      
+    position: relative;
+  }
```

Comment:
It's always a good practice to keep the code clean and organized, I suggest using the standard indentation rules and deleting empty lines.